source/developers-guide/plugin-system/index.md

```diff
@@ -386,6 +386,26 @@ class SwagControllerExample extends Plugin
 }
 ```
 
+### Controller auto-registration
+
+As of version `5.2.7` there is no need for a subscriber which registers your controllers. This will be automatically done by the Shopware core.
+Just place a controller in `SwagControllerExample/Controllers/*` and you are done. 
```

Comment:
you should also mention that a controller must be in a folder like "Frontend" "Backend" etc. just like in the core. otherwise the convention will not work